wire-desktop/PKGBUILD

```diff
@@ -3,25 +3,24 @@
 
 pkgname=wire-desktop
 pkgver=3.5.2881
-pkgrel=1
-pkgdesc='Modern, private messenger. Based on Electron.'
-arch=('x86_64' 'i686')
+pkgrel=2
+pkgdesc='End-to-end encrypted messenger with file sharing, voice calls and video conferences'
+arch=('x86_64')
 url='https://wire.com/'
 license=('GPL3')
-conflicts=('wire-desktop-bin')
-depends=('alsa-lib' 'gconf' 'gtk2' 'libxss' 'libxtst' 'nss' 'xdg-utils')
+depends=('alsa-lib' 'electron' 'gconf' 'gtk2' 'libxss' 'libxtst' 'nss' 'xdg-utils')
```

Comment:
It seems electron pulls in alsa-lib, gtk3, libxss, libxtst, and nss. I don't know if gconf and xdg-utils are needed anymore. I'm currently building with electron as the only dep and I'll see if I can get it to run under docker.

Reply:
xdg-utils is an opt-dep of electron, and I think it is useful for Wire.

Reply:
Can you tell me what will not work without xdg-utils?

Reply:
I removed `gconf` locally, Wire seems to work fine 🤷‍♂️ Updating the list of dependencies to `electron` and `xdg-utils` alone.